roamance/src/components/footer.tsx

```
@@ -0,0 +1,122 @@
+              </li>
+            </ul>
+          </div>
+        </div>
+
+        <div className="border-t mt-12 pt-8 flex flex-col md:flex-row justify-between items-center">
+          <p className="text-sm text-muted-foreground">
+            © {new Date().getFullYear()} Roamance. All rights reserved.
+          </p>
+
+          <div className="flex space-x-4 mt-4 md:mt-0">
+            <a href="#" className="text-muted-foreground hover:text-primary transition-colors">
```

Biome (1.9.4)

[error] Provide a valid value for the attribute href.

The href attribute should be a valid a URL
Check this thorough explanation to better understand the context.

(lint/a11y/useValidAnchor)